### PR TITLE
Added support for python 3.7

### PR DESCRIPTION
**Pull request by vernans**

### CODE REVIEW

**Review by icemac — APPROVED** [review]
LGTM.

File: setup.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
         "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7 ",
```
Comment (mgedmin) [inline setup.py:44]: Are extra trailing spaces in trove classifiers allowed?

Comment (icemac) [reply setup.py:44]: @mgedmin This does not seem to be right, fixed in ac94f16b54e17d78c9093c9a3aec7492576d3162.

Comment (icemac) [reply setup.py:44]: But it does not seem to have hurt in the release. The [PyPI page](https://pypi.org/project/Record/) shows the classifiers correctly.

**Pull request merged**